src/sentry_options.h

```diff
@@ -8,6 +8,12 @@
 #include <godot_cpp/variant/variant.hpp>
 
 class SentryOptions {
+public:
+	enum CaptureType {
```

Comment:
I'm not sure I understand the split between `capture as breadcrumb` and `capture as event`. Those are not mutually exclusive.

Reply:
Makes sense! Thanks for reviewing. I added an issue to track it: #10